test/e2e/config/config.go

```diff
@@ -187,11 +189,23 @@ func (ccc *CustomCloudConfig) SetEnvironment() error {
 	// as azure-cli complains otherwise
 	azsSelfSignedCaPath := "/aks-engine/Certificates.pem"
 	if _, err = os.Stat(azsSelfSignedCaPath); err == nil {
+		// include cacert.pem from python2.7 path for upgrade scenario
```

Comment:
the only difference seems to be the python version

instead of duplicating code the entire block, you may be able to solve it with an extra local variable